### DISCUSSION TITLE
Decentralized Index: Tribler.org

### DISCUSSION THREAD

**Discussion by josecelano** [torrust/torrust-index-backend#418] [Show and tell]
https://www.tribler.org/

If I'm not wrong:

- You can create channels.
- Contents (including metadata) are shared with BitTorrent.
- Metadata is stored in SQLite and also torrents form
- ...

It has a REST API: https://tribler.readthedocs.io/en/latest/restapi/introduction.html

**Comment by synctext** [comment]
Hi !
Tribler founder here. Took us 18 year of work to get here. Tribler as backend should be easy to use with our API. Just a day hopefully.

Lot's of overlap with your Rust indexing project! We're re-writing our Tor- tunneling code into Rust. Python is really slow for such low-level stuff. But for the 15 years we've been experimenting with decentralised trust, it's ideal.
 -j
